modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * cicd/builds/triggering-builds-build-hooks.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to `ClusterRoleBindings`
+
+As a cluster administrator, you can add unauthenticated users to the following ClusterRoleBindings in {product-title}.
```

Comment:
Same about using "cluster role bindings" throughout - I won't point out any more instances in this PR